tests/unit/test-util/test.adb.js

```diff
@@ -1129,6 +1131,44 @@ describe('utils/adb', () => {
       );
     });
 
+    it('informs the user if remote debugging is not enabled', async () => {
+      const adb = getFakeADBKit({
+        adbClient: {
+          shell: sinon.spy(() => Promise.resolve('')),
+        },
+        adbkitUtil: {
+          readAll: sinon.spy(() => {
+            return Promise.resolve(Buffer.from(''));
+          }),
+        },
+      });
+      const adbUtils = new ADBUtils({adb});
+
+      const maxDiscoveryTime = 50;
+      const retryInterval = 10;
+
```

Comment:
Nit, let's remove this two consts and just pass the value in the call to `adb.discoverRDPUnixSocket` (we are not using them for any assertion and so we can make this test a bit shorter by making this small optional change).